packages/app-project/src/shared/components/SubjectPreview/SubjectPreview.js

```diff
@@ -5,84 +5,87 @@ import { FavouritesButton, Media } from '@zooniverse/react-components'
 import { CollectionsButton, TalkLink } from './components'
 
 const defaultSubject = {
-  favorite: false,
-  id: '',
-  toggleFavorite: () => false,
-  locations: []
+	favorite: false,
+	id: '',
+	toggleFavorite: () => false,
+	locations: []
 }
-function SubjectPreview ({
-  height = '250px',
-  isLoggedIn = false,
-  placeholder,
-  slug,
-  subject = defaultSubject,
-  width = '400px'
+function SubjectPreview({
+	height = '250px',
+	isLoggedIn = false,
+	placeholder,
+	slug,
+	subject = defaultSubject,
+	width = '400px'
 }) {
-  const subjectURLs = subject.locations.map(location => Object.values(location)[0])
-  const subjectURL = subjectURLs[0]
-  const collectionsModal = createRef()
-  const href = `/projects/${slug}/talk/subjects/${subject.id}`
+	const subjectURLs = subject.locations.map(location => Object.values(location)[0])
+	const subjectURL = subjectURLs[0]
+	const collectionsModal = createRef()
+	const href = `/projects/${slug}/talk/subjects/${subject.id}`
 
-  return (
-    <Box
-      fill
-    >
-      <Anchor
-        href={href}
-      >
-        <Box
-          align='center'
-          height={height}
-          overflow='hidden'
-          width={width}
-        >
-          <Media
-            alt={`subject ${subject.id}`}
-            height={700}
-            placeholder={placeholder}
-            src={subjectURL}
-            width={700}
-          />
-        </Box>
-      </Anchor>
-      <Box direction='column' gap='xsmall' pad={{ vertical: 'xsmall' }}>
-        <TalkLink
-          href={href}
-        />
-        <FavouritesButton
-          checked={subject.favorite}
-          disabled={!isLoggedIn}
-          onClick={subject.toggleFavourite}
-        />
-        <CollectionsButton
-          disabled={!isLoggedIn}
-          subject={subject}
-        />
-      </Box>
-    </Box>
-  )
+	return (
+		<Box
+			fill
+		>
+			<Anchor
+				role="subject-preview-link"
```

Comment:
This isn't a valid ARIA role, which is why the builds are failing. 

I'd recommend not changing the announced role for links, as that will break them in the accessibility tree.

Reply:
How about using the Title attribute instead? Specifically, I'm wanting to make sure _this particular link_ has the href we're looking for from a testing-perspective.

Reply:
Don't use `title`, particularly on links. It can be confusing in a screen reader and will appear as a tooltip for sighted users. 

https://www.tpgi.com/using-the-html-title-attribute-updated/

Reply:
A CSS class can be used as a selector for a particular class of links.

Reply:
kk - RTL doesn't like class detection. Another alternative is to use the `data-testid` attribute to specifically find an element. Would that me a reasonable compromise?

Reply:
Here's an example from the transcription task, which is tested with RTL, to find SVG groups that represent transcribed lines.

https://github.com/zooniverse/front-end-monorepo/blob/ee66328323a5fb37245f027fad8154bedbe77876/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js#L213-L216

Reply:
The native DOM APIs will always be faster than RTL's queries, so I tend  to use the native APIs where they're suitable. Queries like `getByRole` are wrappers for `document.querySelectorAll`, if you delve into the RTL query code.

Reply:
`data-test-id` works too. CSS classes are good when you want to apply a semantic description to a whole class of elements eg. a list of subject preview links on the home page.

Reply:
You could also apply a class to the preview card itself, then query for `.subject-preview a` to find the link inside a card.

Reply:
Cool - because this is used for testing purposes only I'll opt for the `data-testid` attribute. If this changes in the future it makes sense to turn it into a class / some other query selector.